src/jit/morph.cpp

```diff
@@ -16526,8 +16540,7 @@ void Compiler::fgMorphStmts(BasicBlock* block, bool* mult, bool* lnot, bool* loa
             continue;
         }
 
-#if OPT_MULT_ADDSUB
-
+#if defined(OPT_MULT_ADDSUB) && defined(LEGACY_BACKEND)
         /* Note whether we have two or more +=/-= operators in a row */
```

Comment:
This should be `#if OPT_MULT_ADDSUB && defined(LEGACY_BACKEND)`

`OPT_MULT_ADDSUB` is 0 or 1; `LEGACY_BACKEND` is defined or not defined.

Reply:
Oops, missed that. It looks like OPT_MULT_ADDSUB is pointless it's always 1. The whole thing reduces to `ifdef LEGACY_BACKEND`.